pubsignals/query_test.go

```diff
@@ -114,8 +115,11 @@ func TestCheckRequest_Success(t *testing.T) {
 					v, _ := big.NewInt(0).SetString("17002437119434618783545694633038537380726339994244684348913844923422470806844", 10)
 					return v
 				}(),
-				Operator:            5,
-				Value:               []*big.Int{big.NewInt(800)},
+				Operator: 5,
+				Value: func() []*big.Int {
```

Comment:
not clear why we have to fill values here. In real query it will be still one value

Reply:
because it's not query but `CircuitOutputs`, which always has to be 64 len arr